tests/Test.ADAL.NET.Unit.net45/InteractiveAuthTests.cs

```diff
@@ -169,20 +170,31 @@ public async Task ExtendedLifetimeRetryAsync()
         public async Task AcquireTokenPositiveWithoutUserIdAsync()
         {
             _context = new AuthenticationContext(AdalTestConstants.DefaultAuthorityCommonTenant, new TokenCache());
-            MockHelpers.ConfigureMockWebUI(new AuthorizationResult(AuthorizationStatus.Success,
-                AdalTestConstants.DefaultRedirectUri + "?code=some-code"));
-            AdalHttpMessageHandlerFactory.AddMockHandler(new MockHttpMessageHandler(AdalTestConstants.GetTokenEndpoint(AdalTestConstants.DefaultAuthorityCommonTenant))
+            var mockWebUI = MockHelpers.ConfigureMockWebUI(
```

Comment:
Apart from this unit test, all the Selenium tests now use PKCE. We test AAD, ADFS 2,3 and 4, just like MSAL.